### ISSUE TITLE
508-defect-2 [COGNITION]: How to apply for healthcare Spanish content must have a proper LANG attribute

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21075]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a Spanish speaker who uses assistive technology, I want to hear instructions for applying for healthcare in the correct pronunciation and cadence.

## Details

There is an additional information component on the register for health care page that needs a `lang="es"` attribute. Without this attribute, screen readers attempt to read Spanish content with the English pronunciation dictionary. Context and intent are reduced or lost this way. By adding the proper attribute, screen readers will load and use the Spanish dictionary. Screenshot attached below.

## Acceptance Criteria

- [ ] Lang attribute is added to Spanish content container
- [ ] Screen reader uses the correct inflection and pronunciation. This can be tested by listening to content with a screen reader before the lang attribute is applied, and then again after. There should be a different intonation in the voice announcement, and with correct Spanish pronunciations.

## Environment

* https://staging.va.gov/health-care/how-to-apply/

## Proposed Solution (if known)

```diff
<div
  data-template="paragraphs/spanish_translation"
  data-entity-id="3423"
  class="form-expanding-group additional-info-container form-expanding-group-open"
+ lang="es"
>...</div>
```

## WCAG or Vendor Guidance (optional)

* [Language of Parts: Understanding SC 3.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-other-lang-id.html)

## Screenshots or Trace Logs

![Screen Shot 2021-03-05 at 4 31 45 PM](https://user-images.githubusercontent.com/934879/110181345-2fe36280-7dd1-11eb-9ce8-47d6955bfba4.png)

**Comment by 1Copenut** [comment]
@brianalloyd I found this issue while auditing a VAMC for content. Please feel free to reassign if I've got it in the wrong spot. Thank you!

**Comment by julieadair1** [comment]
@dillo - is this something that still needs to be addressed? 
cc: @jparker0425

**Comment by laflannery** [comment]
This is still something that needs to be addressed but is this something that would/should be handled in the larger translations initiative? If that's the case is there a FE translations epic we could attached this to so we don't lose track of it?
